lib/inputstreamhelper/widevine/arm.py

```diff
@@ -79,7 +79,7 @@ def select_best_chromeos_image(devices):
     for device in devices:
         # Select ARM hardware only
         for arm_hwid in config.CHROMEOS_RECOVERY_ARM_HWIDS:
-            if '^{0} '.format(arm_hwid) in device['hwidmatch']:
+            if device['hwidmatch'].startswith('^{0}'.format(arm_hwid)):
```

Comment:
This is going to be problematic. The reason why we added the whitespace is because if you match for `^BOB`, you will also get `^BOBBA` and `^BOBBA360`, which are both `x86_64`.

Reply:
Thanks, fixed it.